api/resources/translations/messages-en.properties

```diff
@@ -818,7 +818,7 @@ login.hide_password = Hide password
 login.incorrect = Incorrect user name or password. Please try again.
 login.show_password = Show password
 login.sso = Login with SSO
-login.sso.invalid = You are not enabled for log in using SSO.
+login.sso.user_invalid = Your user is not allowed to log in with SSO. Please contact your administrator or supervisor.
```

Comment:
I liked your wording being very simple and targeted to the user. Just took it a bit further by giving them an action to do.  (Also can you update the Swahili translation too?)